backend/de.metas.monitoring/src/main/java/de/metas/monitoring/adapter/PerformanceMonitoringService.java

```diff
@@ -58,57 +56,22 @@ default void monitorSpan(final Runnable runnable, final SpanMetadata metadata)
 	
 	@Value
 	@Builder
-	class SpanMetadata
+	class Metadata
```

Comment:
if this class grows (like adding more methods) consider extracting it to it's own file

nevertheless, if u are keeping it here, pls make sure it's `static`!

Reply:
modifier `static` is redundant for inner classes of interfaces